docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: "3"
+services:
+  bot:
+    restart: unless-stopped
+    container_name: bot
+    image: containers.pkg.github.com/Webeleon/discord-bot-starter
+    env_file:
+      - bot-variables.env
```

```suggestion
      - .env
```

the current .env file for the project is enough.
it is not commited

---

It still miss the mongo in the docker-compose.
It would looks like something like that

```
 links:
      - mongo

      
  mongo:
    image: "mongo"
    env_file:
      - .env
    expose:
      - "27017:27017"
```

---

It seems to me like it's a bad idea to expose the mongo ports if you're not going to use it externally, but only internally for the bot. 
`links` are also [deprecated](https://docs.docker.com/compose/compose-file/#links). Compose [automatically](https://docs.docker.com/compose/networking/) bundles apps into a network together. By extension of the service having the name`mongo`, it is already available to the bot container as `mongo`